mender-deb-package

```diff
@@ -246,11 +246,14 @@ build_packages() {
 }
 
 copy_packages_to_output() {
+  local binary_only_filter=""
+
+  [[ $DEB_BUILD_TYPE =~ source ]] || binary_only_filter=" -and -not -name *.orig.* "
```

Comment:
But looking at the jobs artifacts I think the filter is not working. In any case I believe we should not filter this way but rather avoid copying them in the first place - see comment above.